packages/flutter/lib/src/material/button.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
this should cause a travis failure.

Reply:
(you can see these kinds of failures locally if you run `dev/bots/test.dart`.)

Reply:
Fixed, will remember to keep running that script from now on